enctests/testframework/utils/outputTemplate.py

```diff
@@ -0,0 +1,102 @@
+
+# This has the following requirements.
+# pip install kaleido plotly pandas jinja2
+
+import os
+import plotly.express as px
+import opentimelineio as otio
+import pandas as pd
+import jinja2
+
+
+
+def processTemplate(test_configs, otio_info):
+  tracks = otio_info.tracks[0]
+  
+  tests = {}
+  alltests = []
+  reportconfig = None
+  for config in test_configs:
+    if "reports" in config:
+      reportconfig = config['reports']
+    
+  if reportconfig is None:
+    print("Failed to find report config. Skipping html export.")
+    exit(0)
+
+  testinfo = {'ffmpeg_version': tracks.name.replace("ffmpeg_version_", "")}
+
+
+  for track in tracks:
+      results = []
+      for ref_name, test_info in track.media_references().items():
+          #merge_test_info = {'name': ref_name}
+          #print(ref_name, test_info.metadata)
+          if ref_name == "DEFAULT_MEDIA":
+              #print("Track:", track)
+              continue
+          #if tracks.name not in test_info.metadata[tracks.name][ref_name]:
+          #    print("Not finding:", test_info.metadata[tracks.name][ref_name])
+          #    continue
+          #enc_info = test_info.metadata[tracks.name][ref_name]# [tracks.name]
+          # for key, enc_data in enc_info.items():
+          #     if key == 'results':
+          #         for arg, value in enc_data.items():
+          #             merge_test_info[arg] = value
+          #     else:
+          #         #print("Something ese:", value)
+          #         merge_test_info['encoder_version'] = key
+          #         for arg, value in enc_data.items():
+          #             merge_test_info[arg] = value
+          #print("testInfo:", test_info.metadata)
```

Comment:
We should perhaps remove commented blocks like this and rather bring it back to life with git if we regret.

Reply:
Should be addressed.